.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,24 @@
+# Release Description
+<!--- Summarize what is being released.  -->
+
+# Release Checklist
+<!--- Fill out the following Release checklist -->
+- [ ] Version number has been updated
+- [ ] Description file updated with New Developers (if applicable)
+- [ ] NEWS.md has been updated and issues linked where appropriate
+- [ ] Ensure all unit tests are passing and Test Coverage is Green on Readme
+- [ ] Ensure CMD Checks are passing and says Passing on Readme
+- [ ] GitHub actions on this PR are all passing
```

Comment:
Do we need this? It should not be possible to merge if GitHub actions are failing.

Reply:
Well...we have been letting PRs slide with failing actions.  Was thinking this one should not have anything let through at the end.